wikidata/entity.py

```diff
@@ -238,6 +247,14 @@ def attributes(self) -> Mapping[str, object]:
     def load(self) -> None:
         url = './wiki/Special:EntityData/{}.json'.format(self.id)
         result = self.client.request(url)
+        if result is None:
+            self.data = None  # Not loaded yet
+            self.state = EntityState.not_loaded
+            return
+        elif result == 'Non-Existent':
+            self.state = EntityState.non_existent
+            return
```

Comment:
`Client.request()` returning `None` does not mean “not loaded yet” but “not able to load (non-existent).”